Gemfile

```diff
@@ -62,4 +62,6 @@ end
 
 gem 'rails-controller-testing'
 
-gem 'devise'
\ No newline at end of file
+gem 'devise'
+
+gem 'iex-ruby-client'
```

Comment:
add new line at the end of file

Reply:
what's the purpose of adding a new line at the end of file?